recipes/rift/meta.yaml

```diff
@@ -0,0 +1,123 @@
+{% set name = "RIFT" %}
+{% set version = "0.0.14.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 37edc652de6787be884b1b638e8f41315b0c9ac4fc11a92f9d61d6530991883d
+
+build:
+  number: 0
+  skip: true  # [win or py>30]
```

Comment:
```suggestion
  skip: true  # [win or py2k]
```

Reply:
@marcelotrevisani, I think you mean to suggest `[win or py3k]`, since this code is python2.7 only (for this release), however I don't see why the `pyXk` syntax is better than `py>XY`, since this code also won't work on `py4k` (in principle) so `py>30` is a more complete definition of requirements8.

*<sub>I suppose we should really just use `[win or py!=27]`, since we haven't tested on py<27...</sub>

Reply:
Yes, exact that! sorry for my mistake